openapi/main.yaml

```diff
@@ -0,0 +1,704 @@
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                    description: Details about the error
+                example:
+                  status: 400
+                  error: Could not parse transaction envelope.
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                example:
+                  status: 500
+                  error: An error occurred while processing this request.
+              example:
+                status: 500
+                error: An error occurred while processing this request.
+  /tss/transactions/build:
+    post:
+      tags:
+        - TSS
+      summary: Create transactions with wallet backend channel accounts as the source
+      description:
+        Create transactions with wallet backend channel accounts as the source, from a list of transactions where each transaction is represented as a list of operations that it contains.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                transactions:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      operations:
+                        type: array
+                        description: list of operations that make up the transaction
+                        items:
+                          type: string
+                          description: base64 operation xdr string
+                      timebounds:
+                        type: integer
+                        description: time in seconds after which this transaction will not be accepted by the network
+                    required:
+                      - operations
+                      - timebounds
+              required:
+                - transactions
+            example:
+              transactions:
+                - opearations:
+                  - "Y6MF7SMT2a2d6pt3i37Xx9"
+                  - "M8GA7SMT2a2d6pt3i37Xx8"
+                  timebounds: 300
+      responses:
+        '200':
```

Comment:
So 200 is returned if all transactions provided in the request were built successfully, and 400 is returned if at least 1 transaction couldn't be built?

Reply:
That's right. It's all or nothing. Either all transactions are built successfully or none of them are, from the client's perspective